tests/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/cosmos/cosmos-sdk/tests
 
-go 1.21
+go 1.22
+
+toolchain go1.22.0
```

Comment:
I think we have to remove this to allow people to use lower versions

Reply:
Yes, this is why it is only in simapp and packages that depends on simapp (tests and starship tests)